### PR TITLE
Update data to WEF 2020 Report 

### PR DESCRIPTION
**Pull request by 4bic**
## Description
Updates the data and resource links to [Global Gender Gap Report 2020](https://www.weforum.org/reports/gender-gap-2020-report-100-years-pay-equality) 

## Type of change

- [x] This change  is an update to data and link to report 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: data/data.csv [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
+Senegal,SEN,2143.936766,4628.137243
+Sierra Leone,SLE,1229.362976,1614.653405
+South Africa,ZAF,9239.829363,15131.35169
+Swaziland,SWZ,6838.753467,11758.10074
+Tanzania,TZA,2131.135139,2784.116074
+Togo,TGO,1103.400617,1835.753215
+Tunisia,TUN,4680.79841,17620.09567
+Uganda,UGA,1385.779816,2578.73306
+Zambia,ZMB,3755.849635,3687.303076
+Zimbabwe,ZWE,1852.106699,2515.70312
```
Comment (kilemensi) [inline data/data.csv]: New line

Comment (4bic) [reply data/data.csv]: Hi @kilemensi , I opted for a fork due to permission restrictions.

Comment (kilemensi) [reply data/data.csv]: Sorry for that @4bic ... I've added you to @CodeForAfrica/code-for-africa team.

**Comment by 4bic** [comment]
Updated on **PR** #34

**Pull request closed without merging**